### PR TITLE
Add verbatim string support

### PR DESCRIPTION
**Pull request by sbarzowski**

### CODE REVIEW

**Comment by sbarzowski** [comment]
@lwander @sparkprime

File: parser.go [hunk 1]
```diff
@@ -346,6 +346,8 @@ func (p *parser) parseObjectRemainder(tok *token) (astNode, *token, error) {
 					kind:        astStringBlock,
 					blockIndent: next.stringBlockIndent,
 				}
+			// TODO(sbarzowski) are verbatim string literals allowed here?
```
Comment (sparkprime) [inline parser.go]: Yes they should be allowed everywhere other string literals are allowed.

Comment (sparkprime) [inline parser.go]: I see there is no test for it and it doesn't work in C++:

```
$ jsonnet -e '{@"foo": 3}'
STATIC ERROR: <cmdline>:1:2-7: Unexpected: VERBATIM_STRING_DOUBLE while parsing field definition
```

**Comment by sparkprime** [comment]
LGTM

File: parser.go (hunk 1 above)
Comment (sparkprime) [inline parser.go]: It does work in import, but there is also no test for that

Comment (sparkprime) [inline parser.go]: See also: https://github.com/google/jsonnet/issues/172

**Pull request merged**